INSTALLATION.md

```
@@ -0,0 +1,337 @@
+
+Now sign in to talawa-admin using the `email` and `password` you used to sign up.
+
+# Testing
+
+It is important to test our code. If you are a contributor, please follow these steps.
+
+## Running tests
+
+You can run the tests for `talawa-admin` using this command:
+
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)